kolibri/content/models.py

```diff
@@ -18,13 +18,12 @@
 from le_utils.constants import content_kinds, file_formats, format_presets
 from le_utils.constants.languages import LANGUAGE_DIRECTIONS
 from mptt.models import MPTTModel, TreeForeignKey
-from mptt.querysets import TreeQuerySet
 
-from .content_db_router import get_active_content_database, get_content_database_connection
 from .utils import paths
 
 PRESET_LOOKUP = dict(format_presets.choices)
 
+CONTENT_SCHEMA_VERSION = '0.6.0'
```

Comment:
May also be worth thinking forwards to the "multiple schemas" scenario (topic tree vs _____), in case that influences how we do things now.

For instance, imagine we're moving from Kolibri 0.19 to 0.20, and we add a new kind of channel, say "world map", which uses different models to represent the underlying metadata. Obviously channels of this type will require 0.20+, but we may at the same time still be publishing topic tree channels for the new version that are backwards compatible with 0.19 and below.

Or maybe, for backwards compatibility and for things like coach reports, we'll always have that topic tree in there, and the different channel types just add additional data structures that allow it to be navigated in a different way. But that may not be as adaptable to radically different types of content, such as game worlds or wiki sites.

Not a blocker for 0.6, but perhaps let's do a brainstorming sesh soon?